src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .map(age -> age >= 30 && age <= 45)
+                .orElse(false);
```

Comment:
디미터 법칙과 메서드 참조를 활용할 수 있을 것 같아요.

```suggestion
        return Optional.ofNullable(user)
                .map(User::isSenior)
                .orElse(false);
```